relayer/txmgr/metrics.go

```diff
@@ -13,4 +13,25 @@ var (
 		Name:      "resend_total",
 		Help:      "The total number of transaction resends to a destination chain",
 	}, []string{"chain"})
+
+	txL1GasFee = promauto.NewGaugeVec(prometheus.GaugeOpts{
+		Namespace: "relayer",
+		Subsystem: "txmgr",
+		Name:      "tx_fee_gwei",
+		Help:      "L1 gas fee for transactions in GWEI",
+	}, []string{"chain"})
+
+	txFees = promauto.NewCounterVec(prometheus.CounterOpts{
+		Namespace: "relayer",
+		Subsystem: "txmgr",
+		Name:      "tx_fee_gwei_total",
+		Help:      "Sum of fees spent for all transactions in GWEI",
+	}, []string{"chain"})
+
+	txConfirmationLatency = promauto.NewGaugeVec(prometheus.GaugeOpts{
+		Namespace: "relayer",
+		Subsystem: "txmgr",
+		Name:      "tx_confirmation_latency",
+		Help:      "Latency between transaction submission and confirmation in seconds",
+	}, []string{"chain"})
```

Comment:
I'm not sure about the nature of this latency, but I'm used to always use histograms for durations. This way we can visualize/calculate the 99th percentile latency, and remove outliers from the graphs (better for alerting). I'm not sure if we care about that. 
In addition to that with a histogram we can display a heatmap graph.
But I'm not sure if that's applicable here. I'm happy to show an example.

Reply:
So yes, normally latencies should be a histogram. Especially if the individual measurement are indiepedent of each other. Your HTTP request and my HTTP request is concurrent and unrelated.

But in blocks chain, we do things sequentially. One after the other. In this case, txs are sent and then confirmed one after the other. So the lag/latency is correlated. The second cannot confirm until the first is confirmed.

So a gauge is fine and simpler in this case.

Reply:
Understood and I agree about that - the current value is enough to monitor the system health as a whole. But still in alerts we can track 95th or 99th percentile latency and ignore sporadical outliers.